spark2/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -156,10 +156,13 @@
     this.io = io;
     this.encryptionManager = encryptionManager;
     this.caseSensitive = caseSensitive;
-
-    this.batchReadsEnabled = options.get("vectorization-enabled").map(Boolean::parseBoolean).orElseGet(() ->
-        PropertyUtil.propertyAsBoolean(table.properties(),
-            TableProperties.PARQUET_VECTORIZATION_ENABLED, TableProperties.PARQUET_VECTORIZATION_ENABLED_DEFAULT));
+    boolean batchReadsSparkSessionConf = SparkSession.active().sparkContext().getConf()
+            .getBoolean("read.parquet.vectorization.enabled", true);
+    boolean batchReadsEnabledTableProp = options.get("vectorization-enabled").map(Boolean::parseBoolean).orElseGet(() ->
+            PropertyUtil.propertyAsBoolean(table.properties(),
+                    TableProperties.PARQUET_VECTORIZATION_ENABLED,
+                    TableProperties.PARQUET_VECTORIZATION_ENABLED_DEFAULT));
```

Comment:
Looks like these 4 lines didn't actually change. Can you revert the whitespace changes here?

Reply:
The variable name was changed which caused the previous line to exceed the 120 character limit.